packages/devtools_app/lib/src/memory/flutter/memory_heatmap.dart

```diff
@@ -0,0 +1,497 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+import 'dart:ui';
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart' hide TextStyle;
+import 'package:flutter/rendering.dart' hide TextStyle;
+import 'package:flutter/widgets.dart' hide TextStyle;
+import 'package:intl/intl.dart';
+
+import 'memory_graph_model.dart';
+
+class FlameChart extends StatelessWidget {
```

Comment:
Add a top level comment calling out how this flame chart differs from the other flame chart we support. The main difference I'm aware of is the difference in the navigation scheme. It could make sense to in the future have an option to toggle between the two navigation schemes.

Reply:
Done.